app/models/project.rb

```diff
@@ -1,10 +1,13 @@
 class Project < ActiveRecord::Base
+  before_save :set_github_repo
   belongs_to :organization
   acts_as_ordered_taggable
   acts_as_ordered_taggable_on :technologies, :causes
 
-  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved
-  validates_presence_of :name, :github_repo
+  attr_accessor :git_url
+  attr_accessible :organization_id, :name, :url, :github_repo, :description, :notes, :cause_list, :technology_list, :is_active, :is_approved, :git_url
+  validates_presence_of :name, :git_url
```

Comment:
It would be great if you could keep the `:github_repo` presence validation alongside a more specific `:git_url` presence validation by using the `:on` option: http://guides.rubyonrails.org/active_record_validations.html#on. (Contact me if you have any questions about what I mean by this!)